Disher/View Controllers/LoginViewController.m

```diff
@@ -95,4 +97,10 @@ - (void) registerUser {
     }];
 }
 
+- (BOOL)textFieldShouldReturn:(UITextField *)textField {
+    [self didTapLogin:nil];
+    return YES;
+}
```

Comment:
recommendation: split your PRs to prevent mixing changes. for example, this change to the login fields is not really related to saving recipes/lists into Parse. usually smaller/more constrained PRs are easier to test and to review.